tests/test_utils.py

```diff
@@ -61,17 +62,26 @@ def make_comparible(text):
 
     def test_end_alignment(self):
         encoded = self.encoder.encode_multi_input([self.text], max_length=2000)
-        self.assertEqual(encoded.token_ends[-2], len(self.text.rstrip()))
+        self.assertEqual(encoded.token_ends[-2], len(self.text))
 
 class TestGPT2Encoder(TestGPTEncoder):
     Encoder = GPT2Encoder
 
 class TestRobertaEncoder(TestGPTEncoder):
     Encoder = RoBERTaEncoder
 
+class TestRobertaEncoder(TestGPTEncoder):
```

Comment:
Duplicated class name?

Reply:
Good catch - this is supposed to be TestRobertaEncoderSlow